lighthouse-core/test/gather/gatherers/trace-elements-test.js

```diff
@@ -319,17 +319,40 @@ describe('Trace Elements gatherer - GetTopLayoutShiftElements', () => {
     expectEqualFloat(total, 2.5);
   });
 
-  it('gets animated node ids without duplicates', () => {
+  it('gets animated node ids with multiple animations', async () => {
+    const connectionStub = new Connection();
+    connectionStub.sendCommand = createMockSendCommandFn()
+      .mockResponse('Animation.resolveAnimation', {remoteObject: {objectId: 1}})
+      .mockResponse('Runtime.getProperties', {result: [{
+        name: 'animationName',
+        value: {type: 'string', value: 'alpha'},
+      }]})
+      .mockResponse('Animation.resolveAnimation', {remoteObject: {objectId: 2}})
+      .mockResponse('Runtime.getProperties', {result: [{
+        name: 'animationName',
+        value: {type: 'string', value: 'beta'},
+      }]})
+      .mockResponse('Animation.resolveAnimation', {remoteObject: {objectId: 3}})
+      .mockResponse('Runtime.getProperties', {result: [{
+        name: 'animationName',
+        value: {type: 'string', value: 'gamma'},
+      }]});
+    const driver = new Driver(connectionStub);
     const traceEvents = [
       makeAnimationTraceEvent('1', 5),
       makeAnimationTraceEvent('2', 5),
       makeAnimationTraceEvent('3', 6),
     ];
 
-    const result = TraceElementsGatherer.getAnimatedElements(traceEvents);
+    const result = await TraceElementsGatherer.getAnimatedElements({driver}, traceEvents);
     expect(result).toEqual([
-      {nodeId: 5},
-      {nodeId: 6},
+      {nodeId: 5, animations: [
```

Comment:
note to self: we should probably de-dupe the animation names on the audit side :)